cmd/podman/root.go

```diff
@@ -84,7 +85,15 @@ func init() {
 
 func Execute() {
 	if err := rootCmd.ExecuteContext(registry.GetContextWithOptions()); err != nil {
-		fmt.Fprintln(os.Stderr, "Error:", err.Error())
+		var message string
+		if err.Cause() == define.OCIRuntimeError {
+			// OCIRuntimeErrors include the reason for the failure in the
+			// second to last message in the error chain.
+			message = strings.TrimSuffix(err.Error(), ": " + define.OCIRuntimeError.Error())
+		} else {
+			message = err.Error()
+		}
```

Comment:
An even simpler version of this would be:

```suggestion
		message := strings.TrimSuffix(err.Error(), ": " + define.OCIRuntimeError.Error())
```

Since `strings.TrimSuffix` returns the string unchanged if the suffix is not found, it's not necessary to guard it with an if statement. Still, I thought that maybe there we'll want to add more transformations in the future. Thoughts? <abbr title="you're not gonna need it">YAGNI</abbr>?

Reply:
Actually, I think I will do one more transformation right now.

Reply:
Now I am doing more transformation, so the if guard is needed.